doc/examples/applications/plot_rank_filters.py

```diff
@@ -401,6 +403,49 @@
 
 plt.tight_layout()
 
+######################################################################
+# The example compares the local threshold with the global threshold in 3D
+
+def normalize_pixels(i):
+    i = 256*(i/np.amax(i))
+    return i.astype(np.uint8)
+
+p8 = data.brain()
+
+p8 = normalize_pixels(p8)
+
+radius = 50
+selem = ball(radius)
```

Comment:
This and many following suggestions are on reoccurring templates copied from the 2D version code; should we change the 2D version accordingly?

Reply:
@xialuoke4062 apologies for the confusion here and elsewhere. The 2D code was written a long time ago, and our coding practices and standards have evolved since then. Therefore, when adding new code, we might be more careful than before and we might prefer newer practices over consistency with the old code.

In any case, **no**, we don't expect you to fix the old code. Indeed it's preferable if you don't, to keep this already big PR from growing even bigger. If the inconsistency bugs you after this is merged, you could submit another, separate PR improving the 2D code. =) But for now, we just want to make sure that any new code is as good as possible!

Thank you!!!

Reply:
> We're debating what the best way to call these things going forward, as `selem` is a specialised term that is not very meaningful to beginners. I would suggest `neighborhood = ball(radius)`.

I realize that seem is almost used every place as a variable name. Do you only want this changed in this website demo file?

Reply:
Yes, only here. You are not responsible for our past mistakes! 😅

Reply:
fixed